src/drivers/nuopc/mediator/med_merge_mod.F90

```diff
@@ -34,19 +34,21 @@ module med_merge_mod
   subroutine med_merge_auto(FBout, FB1, FB1w, fldw1, FB2, FB2w, fldw2, &
```

Comment:
I've been thinking about whether that might be a better way to do it.  There is some cost savings (probably small) to actually have it all in one subroutine.  But I agree it might be cleaner the other way.  I can reimplement at some point and see if it works better.  The one thing that immediately falls out is the order of the merge.  Currently, all Imp fields are merged to an FBExp field, then the merge moves onto the next FBExp field.  If we do it the other way, then the outer loop will no longer be the FBExp fields, that will become the inner loop of the merge.  That probably won't change answers but it will affect how the merge is documented.